gui/boosterView.py

```diff
@@ -27,7 +27,8 @@
 
 
 class BoosterViewDrop(wx.PyDropTarget):
-    def __init__(self, dropFn):
+    def __init__(self, dropFn, *args, **kwargs):
+        super(BoosterViewDrop, self).__init__(*args, **kwargs)
```

Comment:
> If we're not doing what the parent is doing, then why are we inheriting the parent?

We are doing what the parent does. That's the point of the `super`. The parent will initialize and do whatever it needs to do, but doesn't require any parameters to work. We inherit from it because that is what we need to do to get drag and drop support. This is basic inheritance. 

> If we are doing what the parent is doing....then why do we have different parameters?

Because __our__ implementation of it does not require additional parameters. 

I'm contacting you on Slack to find out more information on this. If we do go ahead with this, I would prefer to only have `**kwargs` instead of that and `*args`. From what I read on the interwebs it's the preferred method.